src/Components/BlockWithList/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactNode } from 'react'
+
+import {
+  ETheme,
+  widgetThemes,
+} from '../../Constants/Types/theme.types'
+
+import { generateRenderProps } from '../../Utils/helpers'
+
+import './BlockWithList.scss'
+
+export type Props = {
+  /** Define theme */
+  theme?: ETheme
+  frontSide: ReactNode
+  backSide: ReactNode
+}
+
+const defaultProps: Props = {
+  theme: ETheme.DARK,
+  frontSide: <div>frontside</div>,
```

Comment:
We do not need create a default front side. It should be required.
Regarding backSide, if user add this props, we should use flip effect. If a user does not add backSide props. We should use only 1 side withou flip effect